### DISCUSSION TITLE
Import Dashboard not possible if folder empty

### DISCUSSION THREAD

**Discussion by PhilippDehner** [grafana/grafana#40129] [Dashboards, Panels & Visualizations]
**What happened**:
I created a new empty dashboard folder.

**What you expected to happen**:
I wanted to see a button to import a dashboards via link or json.

**How to reproduce it (as minimally and precisely as possible)**:
Just create a new dashboard folder.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.0.1
- Data source type & version: -
- OS Grafana is installed on: Windows
- User OS & Browser: Firefox
- Grafana plugins: -
- Others: -

**Comment by ivanahuckova** [comment]
Hello @trophi20, this seems like a UX issue. You can always do this via `Create > Import > Options > Folder`. FYI @grafana/grafana-labs-ux-design. @grafana/user-essentials passing to you to decide is you want to convert this to discussion or leave it as issue.

**Comment by PhilippDehner** [comment]
Thanks @ivanahuckova, how can I convert it to a discussion? Maybe it would be better at the discussions place

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
